internal/provider/assessor/openai.go

```
@@ -0,0 +1,595 @@
+)
+
+const (
+	openAIVerifierProvider         = "openai"
+	openAIVerifierMaxItemChars     = 4000
+	openAIVerifierMaxTotalChars    = 32000
+	openAIVerifierMaxResponseBytes = 16 << 20
+	openAIVerifierDefaultTimeout   = 60 * time.Second
+
+	// openAIVerifierSystemPrompt is the fixed system instruction for all verification calls.
+	// Temperature is set to 0 when included, and a strict JSON schema is enforced.
+	openAIVerifierSystemPrompt = `You are a fact-verification assistant. Given a claim, optional temporal validity bounds, and a list of evidence items, determine whether the evidence supports ("entailed"), contradicts ("contradicted"), or is insufficient to assess ("insufficient") the claim within the stated temporal scope.
```

golangci-lint (2.12.2)

[error] const openAIVerifierSystemPrompt is unused

(unused)